package.json

```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/MikaAK/s3-plugin-webpack",
   "dependencies": {
+    "aws-sdk": "~2.0.31",
```

Comment:
I kept this as the same version as what the S3 plugin uses. However, I didn't actually test cloudfront invalidation (I'm not using it in my project).